Makefile

```diff
@@ -64,6 +64,7 @@ unit-test-nocoverage-host: golint-host govet-host
 
 build: golint govet
 	vagrant ssh mon0 -c 'sudo -i sh -c "cd $(GUESTGOPATH); make run-build"'
+	make run
```

Comment:
@erikh from #98 looks like @jainvipin wanted `make run` to be added to `start` target but I see you have added it to `build` target instead. Is this intended?